src/src/rx_main.cpp

```diff
@@ -1163,6 +1165,34 @@ static void checkSendLinkStatsToFc(uint32_t now)
     }
 }
 
+static void debugRcvrLinkstats()
+{
+#if defined(DEBUG_RCVR_LINKSTATS)
+    if (debugRcvrLinkstatsPending)
+    {
+        debugRcvrLinkstatsPending = false;
+
+        // Copy the data out of the ISR-updating bits ASAP
+        // While YOLOing (const void *) away the volatile
+        crsfLinkStatistics_t ls = *(crsfLinkStatistics_t *)((const void *)&crsf.LinkStatistics);
+        uint32_t packetCounter = debugRcvrLinkstatsPacketId;
+        uint8_t fhss = debugRcvrLinkstatsFhssIdx;
+        // actually the previous packet's offset since the update happens in tick, and this will
+        // fire right after packet reception (a little before tock)
+        int32_t pfd = PfdPrevRawOffset;
+
+        // Use serial instead of DBG() because do not necessarily want all the debug in our logs
+        char buf[50];
+        snprintf(buf, sizeof(buf), "%u,%u,-%u,%u,%d,%u,%u,%d\r\n",
```

Comment:
I think we need to be able to enable debug logging like this where just the debug lines for a particular "feature" are printed. But I'm ok with this currently.

Reply:
Yeah like some sort of module-grouped logging defines would be nice if we could work out some major functionality groups to put things in.